pom.xml

```diff
@@ -164,7 +164,7 @@ flexible messaging model and an intuitive client API.</description>
     <typetools.version>0.5.0</typetools.version>
     <protobuf3.version>3.19.6</protobuf3.version>
     <protoc3.version>${protobuf3.version}</protoc3.version>
-    <grpc.version>1.45.1</grpc.version>
+    <grpc.version>1.53.0</grpc.version>
```

Comment:
Looks like we need to upgrade to 1.55.0 to address this CVE: https://nvd.nist.gov/vuln/detail/CVE-2023-32731

Reply:
fixed. thanks :)